.husky/post-merge

```diff
@@ -1,4 +1,4 @@
 #!/bin/sh
 . "$(dirname "$0")/_/husky.sh"
 
-yarn
+npm install
```

Comment:
This should be its own PR tbh, if we decide to replace yarn with npm. Mixing is no fun.

Reply:
But we did decide. On ci and scripts should be npm.

Reply:
Alright, but this looks kinda out of scope for this PR. I'll backport to a new one.